build.sbt

```diff
@@ -724,7 +726,6 @@ lazy val storage = project
   .in(file("storage"))
   .enablePlugins(UniversalPlugin, JavaAppPackaging, JavaAgent, DockerPlugin, BuildInfoPlugin)
   .settings(shared, compilation, assertJavaVersion, kamonSettings, storageAssemblySettings, coverage, release, servicePackaging)
-  .dependsOn(rdf)
```

Comment:
You should be able to remove:
```
javaSpecificationVersion := "1.8"
```
From the kernel, rdf, testkit (?) modules.
Also the worflow for storage should not be triggered anymore if those change